nise/__main__.py

```diff
@@ -526,7 +533,7 @@ def main():
     options = vars(args)
     _, provider_type = _validate_provider_inputs(parser, options)
 
-    if not options.get('start_date'):
+    if not (options.get('start_date') or options.get('static_report_file')):
```

Comment:
Won't this fail if you pass start_date w/o static file? Is that the desired outcome?